lib/fbutil.js

```diff
@@ -16,6 +16,7 @@ exports.fbRef = function(path) {
 };
 
 exports.pathName = function(ref) {
+   if(typeof ref.parent === 'undefined') return ref.path;
```

Comment:
I don't see ref.path in the reference guide. Guessing a better answer here would be `ref.ref.key`, as that should work fine with Query and DatabaseReference instances?